### DISCUSSION TITLE
使用流体装罐机会崩溃

### DISCUSSION THREAD

**Discussion by bigbox116** [IndustrialCraftMC/IndustrialCraft2#166] [【漏洞反馈】Bugs Report]
使用流体装罐机用岩浆装罐流体单元游戏会崩溃

**Comment by modred123** [comment]
[crash-2024-09-28_09.25.13-server.txt](https://github.com/user-attachments/files/17171959/crash-2024-09-28_09.25.13-server.txt)
附带一份崩溃报告，对流体灌装机使用岩浆桶会使游戏崩溃。
